### ISSUE TITLE
[BUG] Unable to roll saving throws in any actor sheet

### ISSUE THREAD

**Issue by jeffwain** [MangoFVTT/fvtt-ready-set-roll-5e#222]
**Description**
There's a bug when rolling saving throws for characters that seems to have been around since the last update in all actors, characters or NPCs

**Steps to Reproduce**
1. Create a new world, activate RSR as the only add-on, add one new character
2. Open the character sheet, click any saving throw to try and roll
3. Nothing happens, console error is thrown.

**Versions**
 - Module: [1.5.2]
 - dnd5e: [2.2.1]
 - Foundry: [10.303]

**Other Modules**
None, this happens when only RSR is installed.

**Console error**
```
Uncaught (in promise) TypeError: undefined. key.split is not a function or its return value is not iterable
[Detected 1 package: ready-set-roll-5e]
    at Object.getProperty (commons.js:2158)
    at Localization.localize (foundry.js:6824)
    at Function.localize (modules/ready-set-roll-5e/src/utils/core.js:28)
    at Function.rollAbilitySave (modules/ready-set-roll-5e/src/utils/roll.js:178)
    at Actor5e._actorRollAbilitySave (modules/ready-set-roll-5e/src/utils/patching.js:96)
```
![image](https://github.com/MangoFVTT/fvtt-ready-set-roll-5e/assets/2727756/d5f7ce03-1bb3-4519-833f-b8a2eab2304e)

**Comment by jeffwain** [comment]
Sounds like the actual issue here may be that the V11 update banner is being thrown even though there's still an update available to 1.5.4?

**Comment by MangoFVTT** [comment]
@jeffwain Correct, Foundry blocks updates because the latest version is for v11, but doesn't update to the latest version for v10 before doing so. You will need to install 1.5.4 manually

**Comment by somanyrobots** [comment]
started to file an issue with this same error - confirming, the fix is just to update RSR to v1.5.4 or higher?

**Comment by MangoFVTT** [comment]
@somanyrobots Correct! v1.5.4 if you are still on Foundry version 10, v2.x.x versions if you are on Foundry version 11
